libs/ui/src/background-script.ts

```diff
@@ -4,14 +4,11 @@
 //  3. get rid of ui/background-script imports
 //  4. extract this packages into separate libs
 
-export type { BackgroundMessage } from './messagers/types/background-message.type';
+export { BackgroundMessageType, E2eMessageType, INITIAL_PASSWORD_HASH, LocalStorage } from 'shelter';
+export type { BackgroundMessage } from 'shelter';
```

Comment:
I think `shelter` lib should have file `background-script.ts)` with seperate exports for background script